Source/launcher/routines/_logic.py

```
@@ -1,22 +1,29 @@
-import game_config.structure
-import game_config as config
-from typing import override
-import urllib.request
-import util.versions
-import util.resource
-import urllib.error
-import urllib.parse
+# Standard library imports
+import atexit
+import copy
```

Ruff (0.8.2)

`copy` imported but unused

Remove unused import: `copy`

(F401)